parsers/lib/config.py

```diff
@@ -16,6 +17,7 @@ def refetch_frequency(frequency: timedelta):
     assert isinstance(frequency, timedelta)
 
     def wrap(f):
+        @wraps(f)
```

Comment:
note: [small best practice](https://realpython.com/primer-on-python-decorators/#a-few-real-world-examples) / allows to access the underlying unwrapped function with `f.__wrapped__`

Reply:
Even if this is the best practice and probably something we should do it should not be done in this PR.

Please submit a separate PR for it as touching the core code that is shared between all there parsers need extra consideration and testing before merging.

Reply:
Sorry about that - I have reverted this change 🙌🏽 

Should I remove it from the `retry_policy` decorator down below as well? I think that that one is being used only in `NTESMO`, but happy to get rid of it there too. 

The motivation was to have access to the raw `__wrapped__` function in the tests in order to not have the mocked sessions overwritten by the decorator.

Reply:
It makes sense but I'm not exactly sure how it will impact things on the backend so I'd rather have it in a separate PR I can test out locally with just these changes.

The best would be to move them both to a separate PR.

Thanks in advance!